Cargo.toml

```diff
@@ -27,29 +27,29 @@ path = "criterion.rs"
 required-features = ["bench", "proptest"]
 
 [dependencies]
-anyhow = "1.0.68"
+anyhow = { version = "1.0.68", features = ["backtrace"] }
```

Comment:
If `backtrace` feature is not required for this PR (I doubt it is), let's remove it from here. I'd like to test this particular change separately.

I know it can be useful, but we had some terrible time with eyre + tracing + opentelemetry (the sequencer would just hang on the `info!` macro).

Reply:
Ah, good catch! I hadn't intended to leave this on. I enabled it for debugging. :D